cluster/core/src/main/resources/reference.conf

```diff
@@ -19,10 +19,3 @@ lagom.cluster {
 
 }
 lagom.defaults.cluster.join-self = off
-
-akka.actor.serialization-bindings {
-  "akka.Done"                 = akka-misc
-  "akka.NotUsed"              = akka-misc
-  "akka.actor.Address"        = akka-misc
-  "akka.remote.UniqueAddress" = akka-misc
-}
```

Comment:
Do we want these bindings into `reference-overrides`? I think having them in `reference.conf` was correct already. See https://github.com/akka/akka/blob/release-2.5/akka-remote/src/main/resources/reference.conf#L54-L61

Reply:
The intent of `reference-overrides.conf` is to override `reference.conf`.

So why do you think it's correct to override them in reference.conf?

Reply:
This PR targets `1.5.0` which uses Akka `2.5.x` that doesn't have those bindings.

In any case, if Akka `2.5.x` had those bindings, it's an idempotent setting: we just enable the binding, we are not changing the value. It wouldn't matter if we add it in `reference.conf` or `reference-overrides.conf`.

Reply:
My point being: if we keep this PR as small as possible and as close as possible to the original PR (this is a backport) it'd be better.

Reply:
But it doesn't make any difference.